layer/nrlf/core/validators.py

```diff
@@ -222,7 +222,7 @@ def _validate_relates_to(self, model: DocumentReference):
         """"""
         if not model.relatesTo:
             logger.log(
-                LogReference.VALIDATOR001, step="relates_to", reason="no_relates_to"
```

Comment:
How come we're changing all these validator logs as part of this PR? (it's not mentioned in the ticket)

Reply:
was going through the logs during the documentation stuff and realised that it was the wrong one so just made a few fixes